packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -659,6 +646,21 @@ export class RunningSummarizer implements IDisposable {
 					if (result.success) {
 						return;
 					}
+
+					// Instead of doing multiple attempts, immediately close the summarizer
```

Comment:
It may be better to change the number of attempts to 1 in the following code above which already alters the # of attempts based on config.
Then after the attempts have completed, close the summarizer if "restart" is set.
```
let totalAttempts =
    this.mc.config.getNumber("Fluid.Summarizer.Attempts") ??
    defaultNumberSummarizationAttempts;
```

Reply:
Oh, that would be way better